src/search.cpp

```diff
@@ -457,7 +460,8 @@ Workload Search::refine(Workload wl) {
                 candidate = wl;
                 Indiv n_indiv = Indiv(metric_t::CENQ, q);
                 Comp n_comp = Comp(n_indiv, op_t::GE, Time(c));
-                TimedSpec new_spec = TimedSpec(n_comp, tspec.get_time_range(), total_time);
+                WlSpec* n_comp_ptr = new Comp(n_comp);
+                TimedSpec new_spec = TimedSpec(n_comp_ptr, tspec.get_time_range(), total_time);
```

Comment:
Same comment as [this](https://github.com/all-things-networking/fperf/pull/27/files#r1484920823)

Reply:
[Search handles pointers more elegantly](https://github.com/all-things-networking/fperf/pull/27/commits/00fe78d47212573296f3abb7df4352785a85d2f3)